src/transformers/pipelines/zero_shot_image_classification.py

```diff
@@ -97,9 +97,6 @@ def __call__(self, images: Union[str, List[str], "Image", List["Image"]], **kwar
                 The maximum time in seconds to wait for fetching images from the web. If None, no timeout is set and
                 the call may block forever.
 
-            tokenizer_kwargs (`dict`, *optional*):
-                Additional dictionary of keyword arguments passed along to the tokenizer.
-
```

Comment:
I don't think this should be removed. `tokenizer_kwargs` is a fairly standard input to `_sanitize_parameters` as a way to control tokenizer behaviour. It would also be breaking for anyone using this in their pipelines. 

We might be able to remove in the tests but it should stay here

Reply:
Hmm could you clarify? `tokenizer_kwargs` was added in #29261 which is not yet in a stable release. Hence removing this argument wouldn't break anything

Reply:
Ah, OK, in this case we can remove!